libraries/botbuilder-storage-mssql/src/mssqlStorage.ts

```diff
@@ -0,0 +1,105 @@
+import { Storage, StoreItem, StoreItems } from 'botbuilder-core';
+import { config as msconfig, ConnectionPool, NVarChar, IResult, Int } from 'mssql';
+import { MSSQLOptions } from './schema';
+
+/**
+ * @module @botbuildercommunity/storage-mssql
+ */
+
+function getConnectionPool(connection): ConnectionPool {
```

Comment:
Possible to move this fucntion to a private class method? It should also be possible to mock those.

Reply:
Rewire can only rewrite top level function, so if this is private, it we can't use rewire easily.